src/ripple/rpc/impl/LookupLedger.cpp

```diff
@@ -25,6 +25,23 @@ namespace RPC {
 
 namespace {
 
+/*  Returns true if the server is stand alone or we think the ledger is
+    historical or we have a recent validated ledger and the ledger's ID is at
+    least 10 larger than the validated. The value 10 is not particularly
+    special here, just some space between the ledgers.
+*/
+bool
+synced (NetworkOPs& netOps, Ledger::pointer ledger)
+{
+    return getConfig ().RUN_STANDALONE ||
```

Comment:
I see the result of these conditions in my debugger with a breakpoint easy enough but I admit the code looks complex. Will make it reader friendly.